Turn terminal transparency off by default.  Based on upstream commit
6390c327aa61b83776a45512ca3db08f04fe4a5d for #257762

diff --git a/src/config/options.inc b/src/config/options.inc
index 481d623..eadd5f7 100644
--- a/src/config/options.inc
+++ b/src/config/options.inc
@@ -826,11 +826,13 @@ static struct option_info config_options_info[] = {
 		"2 is 256 color mode, uses XTerm RGB codes")),
 
 	INIT_OPT_BOOL("terminal._template_", N_("Transparency"),
-		"transparency", 0, 1,
+		"transparency", 0, 0,
 		N_("If we should not set the background to black. This is particularly\n"
 		"useful when we have a terminal (typically in some windowing\n"
 		"environment) with a background image or a transparent background -\n"
-		"it will be visible in ELinks as well. Note that this option makes\n"
+		"it will be visible in ELinks as well (but ELinks document color handling\n"
+		"will still assume the background is black so if you have a bright background\n"
+		"you might experience contrast problems. Note that this option makes\n"
 		"sense only when colors are enabled.")),
 
 	INIT_OPT_BOOL("terminal._template_", N_("Underline"),
